docs/config-description.md

```diff
@@ -445,7 +445,7 @@ This document provides a list of all available Clio configuration properties in
 - **Type**: string
 - **Default value**: `%TimeStamp% (%SourceLocation%) [%ThreadID%] %Channel%:%Severity% %Message%`
 - **Constraints**: None
-- **Description**: The format string for log messages. The format is described here: https://beta.boost.org/doc/libs/1_83_0/libs/log/doc/html/log/tutorial/formatters.html.
+- **Description**: The format string for log messages. The format is described here: https://www.boost.org/doc/libs/1_87_0/libs/log/doc/html/log/tutorial/formatters.html.
```

Comment:
We don't use 1.87 yet (sadly). So i suggest we link to the version we actually use right now 👍